### PR TITLE
Update RunConfiguration with TestCaseFilter info

### PR DESCRIPTION
**Pull request by matei-tm**
I added info on TestCaseFilter usage in RunConfiguration section

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@Mikejo5000 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Mikejo5000** [comment]
@Evangelink @engyebrahim Can you take a look at this PR?

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -166,6 +167,7 @@ The **RunConfiguration** element can include the following elements:
 |**TargetPlatform**|x86|**Omit this whole tag to auto-detect.**<br /><br />This setting defines the architecture to use to run tests. Possible values are `x86`, `x64`, `ARM`, `ARM64`, `S390x`.<br /><br />When autodetecting, the architecture for AnyCPU DLLs may differ based on the runner. For .NET Framework runner (in Visual Studio, or vstest.console.exe in Developer command line), the default is x86. For .NET runner (dotnet test), the default is the current process architecture.<br /><br />|
 |**TreatTestAdapterErrorsAsWarnings**|false|false, true|
 |**TestAdaptersPaths**||One or more paths to the directory where the TestAdapters are located|
+|**TestCaseFilter**|| A filter expression in the format __\<property>\<operator>\<value>[\|\&amp;\<Expression>]__. Pay attention to the boolean operator **\&** that should be used as a HTML entity **\&amp;**. Expressions can be enclosed in paranthesis. For detailed syntax on expression structure read [vstest-docs](https://github.com/microsoft/vstest-docs/blob/main/docs/filter.md). |
```
Comment (Evangelink) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:170]: @Mikejo5000 is it better to reuse the exact same description as this one https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/test/vstest-console-options.md:

image

Comment (matei-tm) [reply docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md]: The description above, in addition to some typos, only shows the boolean "or" operator. The filter expression can be built with any of the logical operators ("and", "or"). It is important to mention that the "and" operator must be expressed as an HTML element.

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 2]
```diff
@@ -166,6 +167,7 @@ The **RunConfiguration** element can include the following elements:
 |**TargetPlatform**|x86|**Omit this whole tag to auto-detect.**<br /><br />This setting defines the architecture to use to run tests. Possible values are `x86`, `x64`, `ARM`, `ARM64`, `S390x`.<br /><br />When autodetecting, the architecture for AnyCPU DLLs may differ based on the runner. For .NET Framework runner (in Visual Studio, or vstest.console.exe in Developer command line), the default is x86. For .NET runner (dotnet test), the default is the current process architecture.<br /><br />|
 |**TreatTestAdapterErrorsAsWarnings**|false|false, true|
 |**TestAdaptersPaths**||One or more paths to the directory where the TestAdapters are located|
+|**TestCaseFilter**|| A filter expression in the format __\<property>\<operator>\<value>[\|\&amp;\<Expression>]__. Pay attention to the boolean operator **\&** that should be used as a HTML entity **\&amp;**. Expressions can be enclosed in paranthesis. For detailed syntax on expression structure read [vstest/docs/filter.md](https://github.com/microsoft/vstest/blob/main/docs/filter.md). |
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:170]:
```suggestion
|**TestCaseFilter**|| A filter expression in the format \<property>\<operator>\<value>[\|\&amp;\<Expression>]. The boolean operator **\&** should be represented by the HTML entity **\&amp;**. Expressions can be enclosed in parentheses. For detailed syntax on expression structure, see [vstest/docs/filter.md](https://github.com/microsoft/vstest/blob/main/docs/filter.md). |
```

Comment (Mikejo5000) [reply docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:170]: I removed underscores for bold, as we don't use this syntax.

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md (hunk 1 above)
Comment (Evangelink) [reply docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md]: @matei-tm that makes sense. Would you mind opening a PR on the other page to update the description?

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off @matei-tm Thanks for the doc update!

**Pull request merged**